Brazilian Portuguese Language Support 🇧🇷 (#553)
InnerSourceCommons/innersourcecommons.org: content/pt-br/_index.md

---
title: "InnerSource Commons"
---

<section class="banner banner-head">
  <div class="container">
    <div class="row">
      <div class="col-lg-12 mx-auto text-center">
        <p class="h1">Bem-vindo à InnerSource Commons</p>
      </div>
    </div>
  </div>
</section>

<section class="section section-first">
  <div class="container">
    <div class="row align-items-center mb-3">
      <div class="col-md-9">
        <img src="/images/logo-big.png" class="img-fluid logo-home pr-5" style="float: left;">
        <p>{{< about-text >}}
        </p>
      </div>
      <div class="col-md-3">
        {{< contact-form mode="vertical-light" >}}
      </div>
    </div>
  </div>
</section>

<section class="section bg-light">
  <div class="container">
    <div class="row text-right">
      <div class="col-md-6">
        <p class="section-title h2">Saiba mais sobre InnerSource</p>

>         <p>Se você é novo no InnerSource ou quer saber mais sobre quem está implementando o InnerSource e por que e como começar, visite a seção de Aprendizado do site. Você encontrará uma variedade de recursos para ajudar a acelerar sua jornada rumo ao InnerSource, incluindo materiais de treinamento, melhores práticas, livros, gravações de eventos passados e estudos de caso.

Reviewer: Se você é novo no InnerSource ou deseja saber mais sobre quem está implementando  InnerSource e o porquê e como começar, visite a seção de aprendizado do site. Você encontrará diversos recursos para ajudar a acelerar sua jornada ao InnerSource, incluindo materiais de treinamento, práticas recomendadas, livros, gravações de eventos anteriores e estudos de caso.

>         <p>Se você é novo no InnerSource ou quer saber mais sobre quem está implementando o InnerSource e por que e como começar, visite a seção de Aprendizado do site. Você encontrará uma variedade de recursos para ajudar a acelerar sua jornada rumo ao InnerSource, incluindo materiais de treinamento, melhores práticas, livros, gravações de eventos passados e estudos de caso.

Author: ```suggestion
        <p>Se você é novo no InnerSource ou deseja saber mais sobre quem está implementando InnerSource e o porquê e como começar, visite a seção de aprendizado do site. Você encontrará diversos recursos para ajudar a acelerar sua jornada ao InnerSource, incluindo materiais de treinamento, práticas recomendadas, livros, gravações de eventos anteriores e estudos de caso.
```

        </p>
        <a href="/learn/" class="btn-link">Saiba mais sobre InnerSource<i class="ti-arrow-right"></i></a>
      </div>
      <div class="col-md-6 mt-4 mb-4 mb-md-0 float-right">
        <img src="/images/community/collaboration.png" class="img-fluid pl-4 pr-4">
      </div>
    </div>
  </div>
</section>

<section class="section">
  <div class="container">
    <div class="row align-items-center">
      <div class="col-md-5 mb-4 mb-md-0">
        {{< image-community >}}
      </div>
      <div class="col-md-6">
          <p class="section-title h2">Junte-se à comunidade</p>

>           <p>Apoiamos praticantes e aqueles que desejam aprender sobre o InnerSource por meio de uma ampla variedade de atividades. Para começar, junte-se à crescente comunidade ISC através do Slack e se apresente. Você encontrará ótimos companheiros para acompanhá-lo em sua jornada pelo InnerSource.</p>

Reviewer: Apoiamos praticantes e aqueles que desejam aprender sobre InnerSource por meio de uma ampla variedade de atividades. Para começar, junte-se à crescente comunidade ISC através do Slack e se apresente. Você encontrará pessoas incríveis para lhe acompanhar em sua jornada InnerSource.

>           <p>Apoiamos praticantes e aqueles que desejam aprender sobre o InnerSource por meio de uma ampla variedade de atividades. Para começar, junte-se à crescente comunidade ISC através do Slack e se apresente. Você encontrará ótimos companheiros para acompanhá-lo em sua jornada pelo InnerSource.</p>

Author: ```suggestion
          <p>Apoiamos praticantes e aqueles que desejam aprender sobre InnerSource por meio de uma ampla variedade de atividades. Para começar, junte-se à crescente comunidade ISC através do Slack e se apresente. Você encontrará pessoas incríveis para lhe acompanhar em sua jornada InnerSource.</p>
```

    </div>
  </div>
</section>

<section class="section">
  <div class="container">
    <div class="row align-items-center">
      <div class="offset-md-2 col-md-4 mb-4 mb-md-0">

        {{< announcements >}}
      </div>
      <div class="col-md-5">
        <p class="section-title h2">InnerSource Commons Foundation</p>
        <p>InnerSource Commons é uma organização sem fins lucrativos 501(c)(3) governada por um conjunto de estatutos corporativos.</p>
        <p><a href="/about/" class="btn-link">Sobre a Fundação <i class="ti-arrow-right"></i></a></p>
      </div>
    </div>
  </div>
</section>

<section class="section-last section">
  <div class="container section-small shadow rounded-lg px-4 bg-light">
    <div class="row align-items-center justify-content-center text-center text-md-left">
      <div class="col-lg-5 col-md-4 mb-4 mb-md-0">
        <a class="twitter-timeline" data-height="500" data-dnt="true" href="https://twitter.com/InnerSourceOrg?ref_src=twsrc%5Etfw"></a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
      </div>
      <div class="col-md-5 offset-md-1">
        <p class="h2 section-title">Fique Conectado</p>

>         <p class="mb-4">A maior parte das ações na comunidade acontece em nosso <a href="https://join.slack.com/t/innersourcecommons/shared_invite/zt-1msf8vcqu-fYEHcyI1l4eSPq6rGprMXA">Canal do Slack</a>. Você também pode se manter informado sobre as notícias, eventos e atividades do InnerSource Commons assinando nossa lista de e-mails abaixo, nos seguindo no <a href="https://twitter.com/InnerSourceOrg">Twitter</a> e no <a href="https://www.linkedin.com/company/innersourcecommons">LinkedIn</a>, e se inscrevendo em nosso <a href="https://www.youtube.com/channel/UCoSPSd6Or4F_vpjo4SmyoEA">Canal do YouTube</a>.</p>

Reviewer: <p class="mb-4">A maior parte das ações na comunidade acontece em nosso <a href="https://join.slack.com/t/innersourcecommons/shared_invite/zt-1msf8vcqu-fYEHcyI1l4eSPq6rGprMXA">Canal do Slack</a>. Você também pode se manter informado sobre as novidades, eventos e atividades da InnerSource Commons assinando nossa lista de e-mails abaixo, nos seguindo no <a href="https://twitter.com/InnerSourceOrg">Twitter</a> e no <a href="https://www.linkedin.com/company/innersourcecommons">LinkedIn</a>, e se inscrevendo em nosso <a href="https://www.youtube.com/channel/UCoSPSd6Or4F_vpjo4SmyoEA">Canal do YouTube</a>.</p>

>         <p class="mb-4">A maior parte das ações na comunidade acontece em nosso <a href="https://join.slack.com/t/innersourcecommons/shared_invite/zt-1msf8vcqu-fYEHcyI1l4eSPq6rGprMXA">Canal do Slack</a>. Você também pode se manter informado sobre as notícias, eventos e atividades do InnerSource Commons assinando nossa lista de e-mails abaixo, nos seguindo no <a href="https://twitter.com/InnerSourceOrg">Twitter</a> e no <a href="https://www.linkedin.com/company/innersourcecommons">LinkedIn</a>, e se inscrevendo em nosso <a href="https://www.youtube.com/channel/UCoSPSd6Or4F_vpjo4SmyoEA">Canal do YouTube</a>.</p>

Author: ```suggestion
            <p class="mb-4">A maior parte das ações na comunidade acontece em nosso <a href="https://join.slack.com/t/innersourcecommons/shared_invite/zt-1msf8vcqu-fYEHcyI1l4eSPq6rGprMXA">Canal do Slack</a>. Você também pode se manter informado sobre as novidades, eventos e atividades da InnerSource Commons assinando nossa lista de e-mails abaixo, nos seguindo no <a href="https://twitter.com/InnerSourceOrg">Twitter</a> e no <a href="https://www.linkedin.com/company/innersourcecommons">LinkedIn</a>, e se inscrevendo em nosso <a href="https://www.youtube.com/channel/UCoSPSd6Or4F_vpjo4SmyoEA">Canal do YouTube</a>.</p>
```
